src/Polly.Core/Hedging/OnHedgingArguments.cs

```diff
@@ -14,36 +14,31 @@ namespace Polly.Hedging;
     /// <summary>
     /// Initializes a new instance of the <see cref="OnHedgingArguments{TResult}"/> struct.
     /// </summary>
-    /// <param name="outcome">The context in which the resilience operation or event occurred.</param>
-    /// <param name="context">The outcome of the resilience operation or event.</param>
+    /// <param name="primaryContext">The outcome of the resilience operation or event.</param>
```

Comment:
I think this needs some more explanation on what the difference is between "primary" and "action".

Reply:
Added remarks and enhanced the docs.